doc/whats_new/_contributors.rst

```diff
@@ -179,3 +179,5 @@
 .. _Guillaume Lemaitre: https://github.com/glemaitre
 
 .. _Tim Head: https://betatim.github.io/
+
+.. _Ralph Urlus: https://github.com/RUrlus
```

Comment:
You don't need this change since you used `:user:` that will directly fetch the corresponding github link.